Inputting/InputDefinitions/AliasedTouchscreenInputDefinition.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Inputting.InputDefinitions
+{
+    /// <summary>
+    /// An aliased touchscreen input is basically a named touchscreen coordinate.
+    /// It gets parsed like a regular <see cref="ButtonInputDefinition"/>,
+    /// but gets executed like an input parsed by a <see cref="TouchscreenInputDefinition"/>.
+    /// The resulting input's effective text will be "touch", with the configured touch coordinates
+    /// being passed via additional data in the form of a 2-tuple <c>(x, y)</c>.
+    /// </summary>
+    public struct AliasedTouchscreenInputDefinition : IInputDefinition
```

Comment:
I don't like having aliases duplicate the code for the actual inputdefinition types. Aliases should wrap an inputdefinition, passing in the aliases input, then changing Original and Displayed text on the returned input